src/frontend/test_runner/tests/testdata/range_scan.yaml

```diff
@@ -28,22 +28,26 @@
   batch_plan: |
     BatchExchange { order: [], dist: Single }
       BatchScan { table: orders_count_by_user, columns: [user_id, date, orders_count], scan_ranges: [user_id = Int64(43)] }
+- before:
+    - create_table_and_mv
+  sql: |
+    SELECT * FROM orders_count_by_user WHERE user_id = 1/0
+  batch_error: 'Expr error: NumericOutOfRange'
```

Comment:
Why it's out of range 🥸

Reply:
IIRC, expr eval returns `Option` and all "error" are mapped to NumericOutOfRange 😂